code/game/machinery/computer/medical.dm

```diff
@@ -534,9 +536,9 @@
 			if(prob(10/severity))
 				switch(rand(1,6))
 					if(1)
-						R.fields["name"] = random_unique_name(R.fields["sex"],1)
+						R.fields["name"] = random_unique_name(R.fields["gender"],1)
```

Comment:
does random_unique_name have gender neutral options?

Reply:
Nope, seems to either give first_names_female or defaults to male names
Although I've noticed there's a first_names global, could create a version that uses that.

Reply:
Implemented a neutral option